GooglePayment/build.gradle

```diff
@@ -28,16 +28,18 @@ android {
     }
 }
 
+def braintreeVersion = '3.0.1-SNAPSHOT'
```

Comment:
Non blocking comment: 

Isn't this the same as the `rootProject` version name? Any advantage to redefining it here?

Reply:
`rootProject.ext.versionName` is for braintree-android-google-payment's version.

This is defining braintree-android dependency version.